sled-agent/src/params.rs

```diff
@@ -333,6 +333,8 @@ pub enum ServiceType {
     Dendrite { asic: DendriteAsic },
     Tfport { pkt_source: String },
     CruciblePantry,
+    NTP { servers: Vec<String>, boundary: bool },
```

Comment:
Nit: This should probably follow conventions for Enum's by saying `Ntp` rather than `NTP`. Clippy may complain as well.